### PR TITLE
Update Kolla container images for Ubuntu Jammy Zed

### PR DESCRIPTION
**Pull request by seunghun1ee**
Tested during Ubuntu jammy Yoga to Zed migration with tempest. No alert was fired at Prometheus.

### CODE REVIEW

**Review by Alex-Welsh — CHANGES_REQUESTED** [review]
New kolla tags need a release note

**Review by Alex-Welsh — DISMISSED** [review]
Thanks, LGTM as long as AIOs pass

**Comment by MoteHue** [comment]
Going to be using these next week given the OVN Jammy AIO passed, promotion is running here: https://github.com/stackhpc/stackhpc-release-train/actions/runs/7759396949/job/21163343101

File: etc/kayobe/kolla-image-tags.yml
Comment (Alex-Welsh) [inline etc/kayobe/kolla-image-tags.yml]: I don't think you need to define any of the tags that are the same as the default openstack tag

Comment (seunghun1ee) [reply etc/kayobe/kolla-image-tags.yml]: Should I remove all `ubuntu-jammy` entries that are same to default tag and leave only `rocky-9` tags?

Comment (Alex-Welsh) [reply etc/kayobe/kolla-image-tags.yml]: Yes

Comment (Alex-Welsh) [reply etc/kayobe/kolla-image-tags.yml]: Actually, it may be worth waiting until Will's PR can merge, since it will conflict with this one
https://github.com/stackhpc/stackhpc-kayobe-config/pull/910

Comment (seunghun1ee) [reply etc/kayobe/kolla-image-tags.yml]: It's done

**Review by Alex-Welsh — APPROVED** [review]

**Review by markgoddard — APPROVED** [review]

**Pull request merged**
